maps/away/generic/civilian_station.dm

```diff
@@ -2,7 +2,7 @@
 	name = "civilian station"
 	description = "A modestly-sized independently-owned civilian space station. Many of these exist all throughout inhabited space, offering a place to rest, food to eat, shopping, and refueling - part mall, part motel. This one appears to have been active in the region long before Biesel took control, and an information lookup indicates that it is operated by a small company that is Solarian in origin. This one's transponder says it's open for business!"
```

Comment:
If available in New Ankara, probably shouldn't remain fluffed as Solarian in origin.

Reply:
Addressed